usaspending_api/disaster/management/sql/disaster_covid19_file_d1_awards.sql

```diff
@@ -5,7 +5,7 @@ SELECT
     "transaction_fpds"."referenced_idv_agency_desc" AS "parent_award_agency_name",
     "awards"."parent_award_piid" AS "parent_award_id_piid",
     DEFC."disaster_emergency_funds" AS "disaster_emergency_fund_codes",
-    DEFC."gross_outlay_amount_by_award_cpe" AS "outlayed_amount_funded_by_COVID-19_supplementals",
+    DEFC."gross_outlay_amount_by_award_cpe" + DEFC."ussgl487200_down_adj_pri_ppaid_undel_orders_oblig_refund_cpe" + DEFC."ussgl497200_down_adj_pri_paid_deliv_orders_oblig_refund_cpe" AS "outlayed_amount_funded_by_COVID-19_supplementals",
```

Comment:
I saw in another PR that these 3 columns have a lot of null values; should each of these be COALESCEd?

Reply:
@brett-varney, these columns are being selected from a subquery at the bottom of the SQL. They are coalesced in that select statement, so it is not necessary in this calculation